### DISCUSSION TITLE
Is it possible to pre-scroll the list when the initial value is set?

### DISCUSSION THREAD

**Discussion by pi-chan** [sa-si-dev/virtual-select#133] [Q&A] [answered]
Is it possible to pre-scroll the list when the initial value is set?

For example, if I have option1 to option1000 and I set the selected value to option500, it would be nice to see option500 when the user click on the select input.

Like this,

```js
const options = [1, 2, ..., 1000].map((value) => {
  return {label: `option${value}`, value}
})
VirtualSelect.init({
  ele: element,
  selectedValue: 500,
  options
});
```

**Comment by sa-si-dev** [comment]
As of now, it is not possible. I will implement it in the future release.

**Reply by pi-chan** [reply]
Thanks for the answer.
I tried to do it myself, but it was difficult.
Looking forward to seeing this feature released!

**Comment by sa-si-dev** [comment, marked as answer]
@pi-chan This feature has been implemented in the [release v1.0.29](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.29).

**Reply by pi-chan** [reply]
Thanks for implementing it. I'll try it later.

**Reply by pi-chan** [reply]
It worked perfectly, thank you very much.
